app/src/main/java/de/xikolo/managers/SecondScreenManager.java

```diff
@@ -63,7 +63,67 @@ public void onWebSocketMessageEvent(WebSocketManager.WebSocketMessageEvent event
                     sectionId = message.payload.get("section_id");
                     courseId = message.payload.get("course_id");
 
-                    courseManager.requestCourseWithSections(courseId, courseCallback(message));
+                    NetworkStateLiveData courseRequestNetworkState = new NetworkStateLiveData();
```

Comment:
It would be fine for me to remove the whole `SecondScreen` feature. It is currently broken on the backend and the legacy code is awful. What do you think?

Reply:
I can do that. Should I do it in this PR or open a new one?

Reply:
Best would be to do that first in a PR and rebase this one after merge.